CHANGES.txt

```diff
@@ -27,6 +27,10 @@ Breaking Changes
 Changes
 =======
 
+ - Introduce support for single row subselects in ANY, e.g.::
+
+   Select id from t where id = ANY(Select id from t2)
```

Comment:
plz use capitals for the keywords: SELECT FROM WHERE and indent by 2 spaces from the parent line